packages/hub/src/components/layout/RootLayout/UserControls.tsx

```diff
@@ -17,6 +19,29 @@ import {
 import { chooseUsernameRoute, userRoute } from "@/routes";
 import { DropdownWithArrow } from "./DropdownWithArrow";
 import { DropdownMenuLinkItem } from "@/components/ui/DropdownMenuLinkItem";
+import {
+  DISCORD_URL,
+  GITHUB_DISCUSSION_URL,
+  NEWSLETTER_URL,
+} from "@/lib/common";
+
+export const MenuLink: FC<{
```

Comment:
I imagine we could ideally reorganize these links and stuff, but I think we should do that later.